consensus/consensus_service.go

```diff
@@ -313,13 +311,6 @@ func (consensus *Consensus) UpdateConsensusInformation() Mode {
 
 	consensus.BlockPeriod = 5 * time.Second
 
-	// Enable aggregate sig at epoch 1000 for mainnet, at epoch 53000 for testnet, and always for other nets.
```

Comment:
so we don't need to enable the aggregate sig all together in an epoch?

Reply:
both way works. in a specific epoch. then we have to wait for that epoch and be on hold. Or we just update the first few nodes with aggregate-sig = false (the potential leaders, leader will accept aggregate signatures once the binary is updated, regardless of the flag, the flag is only controlling whether validator will send aggregated signature). and then update the rest with aggregate-sig = true. With this flag. we could have more control on the behavior if needed anyway. 

Which way do you prefer?